src/render.js

```diff
@@ -0,0 +1,46 @@
+import { createProfileTemplate } from './view/profile-view.js';
+import { createSiteMenuTemplate } from './view/site-menu-view.js';
+import { createSortTemplate } from './view/sort-view.js';
+import { createFilmsTemplate } from './view/films-view.js';
+import { createFilmCardTemplate } from './view/film-card-view.js';
+import { createShowMoreTemplate } from './view/show-more-view.js';
+import { createStatisticsTemplate } from './view/statistics-view.js';
+import { createFilmDetailsTemplate } from './view/film-details-view.js';
+
+const FILMS_COUNT = 5;
+
+const RenderPosition = {
+  BEFOREBEGIN: 'beforebegin',
+  AFTERBEGIN: 'afterbegin',
+  BEFOREEND: 'beforeend',
+  AFTEREND: 'afterend',
+};
```

Comment:
```suggestion
const RenderPosition = {
  BEFORE_BEGIN: 'beforebegin',
  AFTER_BEGIN: 'afterbegin',
  BEFORE_END: 'beforeend',
  AFTER_END: 'afterend',
};
```

Константы из нескольких слови принято разделять через _